packages/@aws-cdk-testing/cli-integ/lib/process.ts

```diff
@@ -90,9 +90,9 @@ class PtyProcess implements IProcess {
     this.process.onData((e) => callback(Buffer.from(e)));
   }
 
-  public onStderr(callback: (chunk: Buffer) => void): void {
-    // in a pty all streams are the same
-    return this.onStdout(callback);
```

Comment:
In a TTY, both `stderr` and `stdout` redirect to the same channel, so this created duplicate output in the logs.